src/mocks/mock-test-result.ts

```diff
@@ -74,7 +74,41 @@ export const mockTestResult = (i: number = 0) =>
     testStationType: TestStationType.atf,
     testerName: 'John Smith',
     testerEmailAddress: 'john.smith@dvsa.gov.uk',
-    additionalNotesRecorded: 'notes for the test record will be displayed here...'
+    additionalNotesRecorded: 'notes for the test record will be displayed here...',
+    testVersion: 'Current',
+    createdByName: 'Jane Doe',
+    testHistory: [...createMockList<TestResultModel>(5, (i: number) => mockTestResultArchived(i))]
   });
 
+export const mockTestResultArchived = (i: number = 0) => {
+  const now = new Date();
+  const createdAt = now.setDate(now.getDate() + i);
+  return createMock<TestResultModel>({
+    testResultId: `archivedTestResultId${String(1 + i).padStart(4, '0')}`,
+    createdAt: new Date(createdAt).toISOString(),
+    systemNumber: 'SYS0001',
+    vin: 'XMGDE02FS0H012345',
+    vrm: 'KP02 ABC',
+    testStartTimestamp: new Date().toISOString(),
+
+    testTypes: [...mockTestTypeList()],
+    createdByName: `Person ${i}`,
+    trailerId: `C${String(i + 1).padStart(5, '0')}`,
+    countryOfRegistration: CountryOfRegistration.GreatBritainandNorthernIreland_GB,
+    euVehicleCategory: EuVehicleCategory.M3,
+    odometerReading: 100,
+    odometerReadingUnits: OdometerReadingUnits.KILOMETERS,
+    reasonForCreation: `reason ${i}`,
+    preparerName: 'Durrell Truck & Van Centre',
+    preparerId: 'CM2254',
+    testStationName: 'Abshire-Kub',
+    testStationPNumber: 'P12346',
+    testStationType: TestStationType.atf,
```

Comment:
Probably not one for this PR but do we maybe want to agree on a model/enum naming convention for them to be a bit more consistent

Reply:
True. Better yet, let the swagger contract drive this by generating the models.